sass/_aesop.scss

```diff
@@ -201,15 +220,26 @@
 	}
 }
 
-.aesop-article-chapter {
-    background-color: #333 !important;
+.aesop-article-chapter:after {
+    background-color: $color-dark-grey;
+    bottom: 0;
+    content: '';
+    left: 0;
+    opacity: .4;
+    position: absolute;
+    right: 0;
+    top: 0;
 }
 
 .aesop-cover-title {
-    color: #fff;
     @include rem(font-size, 42px);
-    padding: 250px 0 20px;
     @include site-width;
+    color: #fff;
```

Comment:
Color literals like `#fff` should only be used in variable declarations; they should be referred to via variable everywhere else.<br>Line should be indented 2 spaces, but was indented 4 spaces